buddyboss/buddyboss-platform
# PROD-952 Remove Google+ from Social Networks profile field

### All Submissions:


Fixes #757 

### How to test the changes in this Pull Request:

1. Go To BuddyBoss Profiles
2. Add new field
3. Confirm Google+ is not present in the list

### Proof Screenshots or Video
(In progress)

### Other information:




### Changelog entry

> Remove Google+ from social networks profile field

**Jira issue** : [PROD-952]

[PROD-952]: https://buddyboss.atlassian.net/browse/PROD-952?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
